azure/src/main/java/sunstone/azure/impl/AzureArmTemplateCloudDeploymentManager.java

```diff
@@ -60,12 +64,35 @@ void deploy(String template, Map<String, String> parameters, String group, Strin
             LOGGER.warn("Azure resource group '{}' already exists! It will be reused and deleted when tests are finished.", group);
         }
 
-        armManager.deployments().define(deploymentName)
+        //.create() doesn't allow status check and fails
+        Accepted<Deployment> acceptedDeployment = armManager.deployments().define(deploymentName)
                 .withExistingResourceGroup(group)
                 .withTemplate(template)
                 .withParameters(parametersFromMap(template, parameters))
                 .withMode(DeploymentMode.INCREMENTAL)
-                .create();
+                .beginCreate();
+
+        // polling wait
+        final Duration pollInterval = Duration.ofSeconds(10);
```

Comment:
Where did the 10 seconds come from? Isn't it too much? Moreover, it is a magic constant. Can it be configured via sunstone property?

Reply:
10 sec was just my first estimate, it may not be enough for long files/ slow connection ... or contrary too much, so property it is :+1: